### ISSUE TITLE
Can't support crome 99 version 

### ISSUE THREAD

**Issue by KryptoKirk** [nftdevs/NFTs-Upload-to-OpenSea#114]
Can't support crome 98 version

**Comment by Niklaus0001** [comment]
Yes, I am facing the same issue. I hope it gets resolved/updated fast

**Comment by husainbombaywala** [comment]
im facing same issue on 5 March 2022

**Comment by Niklaus0001** [comment]
Go to Chrome, search Chromedriver or directly go to https://chromedriver.chromium.org/, then just to confirm go to settings and confirm your chrome version. Once you have found the version just download the driver. Open the ZIP file and the OpenSea Upload folder and over there you will be able to see Chromedriver. Just press and hold on the Chromedriver(in the Zip folder) and take to the OpenSea folder and then you will get a popup saying if you want to replace the file and stuff.
Click on Replace the file.
